packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -179,6 +221,36 @@ export default class QuanticGeneratedAnswer extends LightningElement {
     return !this?.searchStatusState?.hasError && this.state?.error?.isRetryable;
   }
 
+  /**
+   * Returns the options displayed in the Quantic Feedback Modal.
+   */
+  get options() {
```

Comment:
So for now we won't allow customers to give us options for feedback as a param?
Might be an idea for the future?